ui/package.json

```diff
@@ -43,18 +41,11 @@
     "react-scripts/**/postcss-normalize": "10.0.1",
     "**/react-beautiful-dnd": "git+https://github.com/gojekfarm/react-beautiful-dnd.git#master"
   },
-  "lint-staged": {
```

Comment:
This is now performed by the pre-commit prettier hook, minus the auto-staging of modified files. This is detailed in the PR description.